src/components/ProgressBar.scss

```diff
@@ -0,0 +1,70 @@
+// @import "../base/index.scss";
```

Comment:
You can delete this whole file since there's the new progress.scss file now

Reply:
I had deleted that in a previous commit, but I think with the rebasing, we deleted the wrong one yesterday. The same applies to the notes below where I actually changed most of the scaleGrid(x) to $su-size variables so that it would be more consistent.